spec/controllers/home_controller_spec.rb

```diff
@@ -11,7 +11,7 @@
   let(:future) { create :far_future_rental, reservation_id: 7777 }
   let(:ongoing) { create :ongoing_rental, reservation_id: 8888 }
   let(:ongoing2) { create :ongoing_rental, reservation_id: 9999 }
-  let(:canceled) { create :mock_rental, rental_status: 'canceled', reservation_id: 0000 }
+  let(:canceled) { create :mock_rental, rental_status: 'canceled', reservation_id: 00_00 }
```

Comment:
yes, but they were starting at 1111 instead of just 1 thinking there would be other rentals created by `create :mock_rental` which has a `sequence :reservation_id`, therefore to avoid collisions  they started there.